recipes/chartify/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "chartify" %}
+{% set version = "2.3.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3f991749eb85c90bb527eb08f242799325ad0f891f3744c9b2ed653fa54ece50
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - bokeh >=0.12.16
+    - colour >=0.1.5
+    - jupyter >=1.0.0
+    - pandas >=0.21.0
+    - pillow >=4.3.0
+    - python >=3.5
+    - scipy >=1.0.0
+    - selenium >=3.7.0,<=3.8.0
+
+test:
+  imports:
+    - chartify
+
+about:
+  home: https://github.com/spotify/chartify
+  license: Apache 2.0
+  license_file: LICENSE
+  summary: Python library to make plotting simpler for data scientists
+
+extra:
+  recipe-maintainers:
+    - cphalpert
```

Comment:
@cphalpert are you okay with being a maintainer?

Reply:
https://github.com/spotify/chartify/issues/25